includes/rest-api/class-sensei-rest-api-extensions-controller.php

```diff
@@ -133,13 +165,125 @@ function( $plugin ) use ( $params ) {
 			}
 		);
 
+		return $this->create_plugins_response( $filtered_plugins );
+	}
+
+	/**
+	 * Update a single plugin.
```

Comment:
I think it updates multiple plugins, instead of a single one, right?

Reply:
Fixed in 72d93de64dc33988741faaf2ad76689616ce372f